app/i18n/locales/en.js

```diff
@@ -257,6 +257,7 @@ export default {
 	No_Reactions: 'No Reactions',
 	No_Read_Receipts: 'No Read Receipts',
 	Not_logged: 'Not logged',
+	Not_is_rc: 'Apparently this is not a server Rocket.Chat - Contact your server admin',
```

Comment:
I think it should be a little different.
`Not_RC_Server`, `This is not a Rocket.Chat server` and we should try to separate `Contact your server admin` in another sentence, so it appears this way...
```
This is not a Rocket.Chat server.
Contact your server admin.
```